apps/demos/Demos/Stepper/Overview/jQuery/styles.css

```diff
@@ -0,0 +1,104 @@
+.dx-viewport {
+  height: 800px;
+  width: 100%;
+  margin: 0;
+}
+
+.stepper-demo {
+  display: flex;
+  height: 100%;
+  gap: 20px;
+}
+
+.widget-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: calc(100% - 280px);
+  min-width: 320px;
+  overflow: clip;
+}
+
+.widget-wrapper {
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  row-gap: 80px;
+  width: 100%;
+}
+
+.stepper-wrapper {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+}
+
+.widget-wrapper-vertical {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  .stepper-wrapper {
```

Comment:
In demos we don't use nesting because build-in editor doesn't support such syntax on Widget Gallery site.